.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules
+package-lock.json
+*.wasm
```

Comment:
I wasn't done, but normally `.wasm` files would be build output and not committed into to the repo.

Also `package-lock.json` files are ignored by NP for packages. They are only useful for applications (that consume packages).